From e698f31d45ab8595875837ddf23be7b23de83f5e Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Wed, 7 Sep 2016 07:54:27 -0500
Subject: [PATCH 05/26] patches: native - missing Parcel symbol.

-Not quite sure if this is the right way to do this. Since
there are 2 writeString16 functions, which as far as I
can tell have not changed, it's possible that this
symbol represents both functions or the "other" one.

-More research is indicated.
---
 libs/binder/Parcel.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/binder/Parcel.cpp b/libs/binder/Parcel.cpp
index 78b9c21..729d6fd 100644
--- a/libs/binder/Parcel.cpp
+++ b/libs/binder/Parcel.cpp
@@ -377,6 +377,8 @@ status_t unflatten_binder(const sp<ProcessState>& proc,
 
 // ---------------------------------------------------------------------------
 
+extern "C" status_t _ZN7android6Parcel13writeString16EPKtj() __attribute__((alias("_ZN7android6Parcel13writeString16EPKDsj")));
+
 Parcel::Parcel()
 {
     LOG_ALLOC("Parcel %p: constructing", this);
-- 
2.9.3

